rs/nns/governance/src/proposals/install_code.rs

```diff
@@ -141,6 +145,37 @@ impl CallCanister for InstallCode {
     }
 }
 
+impl CallCanister for InstallCode {
+    fn canister_and_function(&self) -> Result<(CanisterId, &str), GovernanceError> {
+        let canister_id = self.valid_canister_id()?;
+        if canister_id != ROOT_CANISTER_ID {
+            return Ok((ROOT_CANISTER_ID, "change_nns_canister"));
+        }
+
+        let install_mode = self.valid_install_mode()?;
+        match install_mode {
+            RootCanisterInstallMode::Install | RootCanisterInstallMode::Reinstall => {
+                Err(Self::invalid_proposal_error(&format!(
+                    "InstallCode mode {:?} is not supported for root canister, consider using \
+                     HardResetNnsRootToVersion proposal instead",
+                    install_mode
+                )))
```

Comment:
No change.

Would it be feasible to add support for this later? I read the PR description, and I understand that not supporting this is intentional, and is the plan going forward, but I'm not really understanding the why. Maybe, I need to re-read the description...

Reply:
I don't see any problem with supporting it, but I just don't see any reason to do so.  Added some comments.